### PR TITLE
Use new stage1 argument when running rkt node.

### LINKED ISSUE #801: calicoctl node probably doesn't pass correct options to rkt
Our guide currently uses rkt 0.16 so we're OK for now, but we need to update before the next release to use the new options - https://github.com/coreos/rkt/pull/1977

### PR DESCRIPTION
**Pull request by tomdee**
Fixes #801

### CODE REVIEW

**Comment by caseydavenport** [comment]
So this will always look for `/usr/local/rkt/stage1-fly.aci`?

How does this work on CoreOS machines where that image isn't provided?  `/usr` is read-only so we can't install `stage1-fly.aci`.

**Comment by tomdee** [comment]
@caseydavenport I've redone this. Can you take another look?

File: docs/cni/rkt/cloud-config/master-config.yaml [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
       # Add /opt/bin to the _front_ of the PATH.
       # Can't directly write to .profile since it's a symlink to a RO filesystem      
       rm /home/core/.bashrc
       echo 'PATH=/opt/bin:$PATH' > /home/core/.bashrc
       echo "Defaults  secure_path=/opt/bin:$PATH" >>/etc/sudoers.d/path
       mkdir -p /opt/bin
-      curl -L https://github.com/coreos/rkt/releases/download/v0.16.0/rkt-v0.16.0.tar.gz | tar --strip-components=1 --overwrite -zxv -C /opt/bin
-      ln -s /opt/bin/stage1-fly.aci /home/core
-      ln -s /opt/bin/stage1-fly.aci /
+      mkdir -p /opt/bin
```
Comment (caseydavenport) [inline docs/cni/rkt/cloud-config/master-config.yaml]: Why move all of this into this script rather than leave it as ExecStartPre options?  Just curious.

Comment (tomdee) [inline docs/cni/rkt/cloud-config/master-config.yaml]: Consistency. Some things in the script can't (or shouldn't) be done in ExexStartPre (e.g. the pipes and redirects).

Comment (caseydavenport) [inline docs/cni/rkt/cloud-config/master-config.yaml]: Makes sense.

**Comment by caseydavenport** [comment]
LGTM once tests pass.

**Pull request merged**